Amazon_Web_Services/RDS/Log-Collection/rds.py

```diff
@@ -29,7 +29,7 @@
 #Get a list of the logs that have been modified since last run
 dbLogs=client.describe_db_log_files(
     DBInstanceIdentifier=dBInstanceIdentifier,
-    FileLastWritten=lastReadDate, # Base this off of last query
```

Comment:
This comma causes the script to crash because there are no more arguments to this function call.

Reply:
Hi Steve, thank you very much for your suggested improvements. We'll review your PR as soon as possible.

Reply:
@stevenchamales Hey - I'm not finding trailing commas to be causing any crashing on my side. Can you provide more details on this to help understand more?

Reply:
@danreichert , my mistake, you are correct; the comma doesn't cause a crash. 
Hope you didn't blow too much time looking into it.  If you did, I owe you a beer.